lib/http.js

```diff
@@ -70,7 +78,7 @@ function buildVerboseError(method, params, response) {
 }
 
 function dumpResponse(response) {
-  const body = (response && response.body && JSON.stringify(response.body)) || response.text;
+  const body = (response && response.data && JSON.stringify(response.data)) || response.text;
```

Comment:
text is not available on axios response.

```suggestion
  const body = (JSON.stringify(response?.data ?? {}));
```